src/wdio/visualcompare.js

```diff
@@ -0,0 +1,37 @@
+import path from 'path';
```

Comment:
Maybe I'm missing it, but where is this file used? There wasn't reference in the README and it didn't seem to have any test cases.

Reply:
Its used in the base config... Its our custom config for the visualregression services. Consumers don't need to do anything with it.

Reply:
OH duh. Yeah I see its the config which was referenced [here](https://github.com/cerner/terra-toolkit/pull/18/files#diff-d668d1220134a102bb565faaf3b911afR3). Can we update this file name to be something like visual_regression.conf.js? it sorta sounds like a helper which why I thought to look in the test cases